docs/tutorials/vehicle_model/README.md

```diff
@@ -0,0 +1,242 @@
+# Tutorial: Create a vehicle model with DTDL
+
+- [Introduction](#introduction)
+- [1. Create a reference vehicle model with DTDL](#1-create-a-reference-vehicle-model-with-dtdl)

```

Comment:
Please capitalize nouns, pronouns, verbs, adjectives, and adverbs in section titles (e.g. the section on this line should be "Create a Reference Vehicle Model with DTDL")